release_creation/bundle/requirements/v1.4.latest.requirements.txt

```diff
@@ -1,6 +1,3 @@
-# Temporary fix to avoid setuptools dependency conflict with dbt-core
-# This is added to get the 1.4.x release published.
-setuptools~=40.8.0
```

Comment:
I think we still need to include setuptools in the requirements

Reply:
I don't see it in any other files. Is there a specific reason 1.4 needs it but no other versions? And then, if so, which version? It appears that 40.8.0 is not current enough.

Reply:
Fantastic. Well, I added it back in, but completely unpinned, so that we get the latest. I think that's a low risk.